bin/node-sass

```diff
@@ -229,63 +229,41 @@ function getOptions(args, options) {
  */
 
 function watch(options, emitter) {
-  var buildGraph = function(options) {
-    var graph;
-    var graphOptions = {
-      loadPaths: options.includePath,
-      extensions: ['scss', 'sass', 'css']
-    };
-
-    if (options.directory) {
-      graph = grapher.parseDir(options.directory, graphOptions);
-    } else {
-      graph = grapher.parseFile(options.src, graphOptions);
-    }
+  var handler = function(files) {
+    files.added.forEach(function(file) {
+      var watch = gaze.watched();
```

Comment:
Sorry I'm still not following. 

If I'm understanding correctly wouldn't that then couple us to the fs watcher implementation's api? 
What happens if the next fs watcher doesn't expose a method like `watch()` for exposing all currently watched files?

The `watch.indexOf(file) === -1` is an optimisation to prevent calling `gaze.add` superfluously. Under the covers `gaze.add` also does deduping but there's not guarantee the next fs watcher would.